src/components/Bakeries/Schedule/ScheduleContainer.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router";
+import Schedule from "./Schedule";
+import { mainApi } from "../../../api/api.js";
+
+const ScheduleContainer = (props) => {
+  let param = useParams();
+  const [schedule, setSchedule] = useState([]);
+  const [name, setName] = useState("");
+  const [err, setError] = useState(false);
+
+  useEffect(() => {
+    (async function () {
+      let map = new Map(),
+        schedule = [],
+        prevDayName = "",
+        list = await mainApi.getBakery(param.name);
+
+      if (list.message) {
+        setError(true);
+      } else {
+        let upperFirst = (day) =>
```

Comment:
it's possible to do with CSS :)

Reply:
I don't have an idea), we can write kind of the same condition in visual component right in jsx part of code, but I don't like this style. I prefer to make a clean visual component.

Reply:
Hehe, yep, right tool in right place quite important
https://css-tricks.com/almanac/selectors/f/first-letter/

Reply:
I did it wrong, and I noticed that it wasn't in a test. So I just was lazy to make it with small letters like in a screen example. But, at least now I've known something new.